src/plugins/GUITestBase/src/GTUtilsDashboard.cpp

```diff
@@ -24,17 +24,78 @@
 #include <primitives/GTWidget.h>
 
 #include <QRegularExpression>
+#include <QStringBuilder>
 #include <QTabWidget>
 
 #include <U2Gui/HoverQLabel.h>
 
 #include "GTUtilsDashboard.h"
 
+namespace {
+
+// A substring with the name of the notification type is searched for. If the |str| contains exactly one type of
+// notification, then this type is returned, otherwise a null QVariant is returned.
+QVariant getNotificationTypeFromString(const QString &str) {
+    using Notification = U2::GTUtilsDashboard::Notification;
+
+    QVariant returnable;
+    constexpr int typesNumber = sizeof(Notification::typesNames) / sizeof(QString);
+    int counter = 0;
+    Notification::Type type;
+
+    for (int typeInd = 0; typeInd < typesNumber; ++typeInd) {
+        if (str.contains(Notification::typesNames[typeInd], Qt::CaseInsensitive)) {
+            ++counter;
+            type = static_cast<Notification::Type>(typeInd);
+        }
+    }
+
+    if (counter == 1) {
+        returnable = static_cast<int>(type);
+    }
+    return returnable;
+};
+
+#define GT_CLASS_NAME "GTUtilsDashboard"
+
+#define GT_METHOD_NAME "getCellText"
+// Returns text from the (row, column) table cell. The (row, column) cell must exist
+QString getNotificationCellText(HI::GUITestOpStatus &os, const QGridLayout &tableLayout, const int row, const int col) {
+    QString text;
+    const QWidget *const cellWidget = tableLayout.itemAtPosition(row, col)->widget();
```

Comment:
nit: too many local consts make the code less readable.

Reply:
Same here: you introduce very verbose style that is new and unusual to UGENE.
When it comes to local variables in a simple 10-line method 2 consts in variable type declaration makes the code less readable.

I see the following simple solutions:
- Want to avoid such comments: follow the common style found across other files in UGENE.
- Want to make this style to be a standard: write a email with a suggestion to the team alias before you use it.

In either case this comment is a 'nit' - not a blocker for the final approval